### PR TITLE
remove ingress resources and use another config example

### PR DESCRIPTION
**Pull request by adamancini**

### CODE REVIEW

File: manifests/example-configmap.yaml [hunk 1]
```diff
@@ -1,7 +1,32 @@
...
+    <!doctype html>
+    <html>
+      <head>
+        <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
+        <title>Demo Application</title>
+        <link href="styles.css" rel="stylesheet" type="text/css">
+      </head>
+      <body>
+        Hello from KOTS.
+        <p>This is text from a user config value: '{{repl ConfigOption "example_defult_value"}}' </p>
```
Comment (dexhorthy) [inline manifests/example-configmap.yaml:17]: nice

File: manifests/kots-config.yaml [hunk 2]
```diff
@@ -1,20 +1,46 @@
...
+    title: My Example Config
+    description: Configuration to serve as an example for creating your own. See [https://kots.io/reference/v1beta1/config/](https://kots.io/reference/v1beta1/config/) for configuration docs. In this case, we provide example fields for configuring an Nginx welcome page.
+    items:
+    - name: show_text_inputs
+      title: Customize Text Inputs
+      help_text: "Show custom user text inputs"
+      type: bool
+      default: "0"
+      recommended: true
+    - name: example_defult_value
```
Comment (jdewinne) [inline manifests/kots-config.yaml:18]: How about `example_default_value` instead of `example_defult_value`?

**Review by jdewinne — APPROVED** [review]
See single comment (typo), but wouldn't mind merging including the typo (can be fixed later)

File: manifests/kots-config.yaml (hunk 2 above)
Comment (adamancini) [reply manifests/kots-config.yaml:18]: fixt

**Pull request merged**